### ISSUE TITLE
GO:0004104: inexact synonyms tagged as exact

### ISSUE THREAD

**Issue by rwst** [geneontology/go-ontology#18928]
The synonyms of cholinesterase listed below don't seem exact synonyms but rather are narrow:
```
id: GO:0004104
name: cholinesterase activity
namespace: molecular_function
def: "Catalysis of the reaction: an acylcholine + H2O = choline + a carboxylic acid anion." [EC:3.1.1.8]
subset: goslim_chembl
synonym: "anticholineesterase activity" EXACT [EC:3.1.1.8]
synonym: "benzoylcholinesterase activity" EXACT [EC:3.1.1.8]
synonym: "BtChoEase activity" EXACT [EC:3.1.1.8]
synonym: "butyrylcholine esterase activity" EXACT [EC:3.1.1.8]
synonym: "butyrylcholinesterase activity" EXACT [EC:3.1.1.8]
synonym: "propionylcholinesterase activity" EXACT [EC:3.1.1.8]
synonym: "pseudocholinesterase activity" EXACT [EC:3.1.1.8]
```
Can you confirm?

**Comment by ukemi** [comment]
Seems correct, they appear to all be more specific and were derived from the EC record. @hdrabkin, sanity check?

**Comment by hdrabkin** [comment]
They are correct in EC; this must stem from the original population of the EC where the synonyms went in automatically as exact (note how low the id is!). However, *I* would label them narrower. If there are already more specific terms in GO for these I can get the RHEAs. 
Clearly this term could be used at present with input/outputs. I can change these to narrow synonyms.
Currently 116 annotations in amigo, 38 experimental
The single child GO:0003990  acetylcholine esterase activity; 59 annotations, 29 experimental

I don't think it's worth making the actual subterms at this time though.

**Comment by hdrabkin** [comment]
Here is what EC says
Hi Harold,

I am not sure I understand the difference between 'exact synonym' and 'related'
in GO terminology. In IUBMB terminology alternative names are synonyms, related
names would belong to related enzymes that should have their own EC numbers.
Generally, there is no fixed policy concerning synonyms in the EC list, it is
up to the person doing the transfer to decide whether he or she wants to keep
the existing synonyms or not. A transfer often means that what was originally
thought to be the activity of the enzyme is not correct. So enzymes linked to
the 'original' function stop making sense as well. In such a case we do not
keep the original synonym.
In addition, if you get your EC data from ENZYME, we do not show all the
synonyms listed by IUBMB, as IUBMB synonyms include gene names and old versions
of the IUBMB systematic name. the IUBMB systematic name is not part of ENZYME
data, and gene names are shown and maintained in UniProtKB so we do not like to
duplicate it in ENZYME where is it not certain to be up-to-date.

Best,
Kristian

So bottom line is we should maybe not assume if EC uses it as an alternative name that is is an exact synonym.

**Comment by hdrabkin** [comment]
The synonyms were changed to 'related'.  I will leave them as 'related' and close this ticket.
